ui/app/templates/vault/cluster/access.hbs

```diff
@@ -7,14 +7,14 @@
         {{/link-to}}
       </li>
     {{/if}}
-    {{#if (has-permission "access" routeParams="entities")}}
+    {{#if (has-permission "access" routeParams="entities" capabilities=(array "list"))}}
```

Comment:
Should this get passed from the template? It seems like the other required things are defined in the map in the service. Maybe making the value an array in stead of a string like : `entities: ['identity/entity/id, 'list'],`

Reply:
that's a great point. i do like the idea of coupling the required capability with each path, although so far it's only necessary for the entity and groups pages, and the refactoring turned out to be tricky. i tried to find a happy medium in [this commit](https://github.com/hashicorp/vault/pull/7590/commits/53b4920bbf96bf12644183eb22e107b4cb046bfe) -- open to feedback!